src/components/molecules/Visualizer/Engine/Cesium/Tileset/index.tsx

```diff
@@ -41,11 +43,18 @@ export default function Tileset({ layer }: PrimitiveProps<Property>): JSX.Elemen
       setStyle(new Cesium3DTileStyle(await res.json()));
     })();
   }, [styleUrl]);
-
-  return !isVisible || !tileset ? null : (
+  const tilesetUrl = useMemo(() => {
+    return sourceType === "osm" && isVisible
+      ? IonResource.fromAssetId(96188)
```

Comment:
Please leave a comment here at least mentioning that this is will retrieve the OSM URL and/or a link to the code on github that you are referencing for this. Makes it easier for other devs to know what this is about.